urlshortener/db.py

```diff
@@ -0,0 +1,44 @@
+import azure.cosmos.cosmos_client as cosmos_client
+import azure.cosmos.exceptions as exceptions
+from azure.cosmos.partition_key import PartitionKey
+from os import environ
+
+
+def setup_db():
+    HOST = environ["COSMOS_HOST"]
+    MASTER_KEY = environ["COSMOS_MASTER_KEY"]
+    DATABASE_ID = "urlshortener"
+    CONTAINER_ID = "links"
+    CONTAINER_ID2 = "users"
+    container = {}
+
+    client = cosmos_client.CosmosClient(
+        HOST, {'masterKey': MASTER_KEY},
+        user_agent="CosmosDBPythonQuickstart", user_agent_overwrite=True)
+
+    try:
+        db = client.create_database(id=DATABASE_ID)
+        print('Database with id \'{0}\' created'.format(DATABASE_ID))
+    except exceptions.CosmosResourceExistsError:
+        db = client.get_database_client(DATABASE_ID)
+        print('Database with id \'{0}\' was found'.format(DATABASE_ID))
+
+    try:
+        container["links"] = db.create_container(
+            id=CONTAINER_ID,
+            partition_key=PartitionKey(path='/partitionKey'))
```

Comment:
I suspect you can simplify to use a single container, but for the sake of instruction: rather than using a dict, you can use an object with defined slots:
```
class Containers(object):
    __slots__ = ('links', 'users')
```
This will allow you to reference `containers.links` but will error on `containers.foo`.